### ISSUE TITLE
May I ask why I can't download mediapipe and keep going in circles

### ISSUE THREAD

**Issue by wsjhou** [nonnonstop/sd-webui-3d-open-pose-editor#35]

**Comment by nonnonstop** [comment]
which version?
![image](https://user-images.githubusercontent.com/42905588/229267205-2fa372f7-8678-4338-8fe9-83819f7a9e86.png)

**Comment by dsickorez** [comment]
version 99f3e80d (Tue Mar 28 11:58:16 2023) here and also hangs on downloading the model.  Would appreciate alternate download / install instructions for pose detector in the meantime.

**Comment by nonnonstop** [comment]
Are there any files in `extensions\sd-webui-3d-open-pose-editor\downloads\pose\0.5.1675469404`?
The files are automatically downloaded by webui.sh/webui.bat.

**Comment by Jseca666** [comment]
me too!

**Comment by wsjhou** [comment]
> Are there any files in `extensions\sd-webui-3d-open-pose-editor\downloads\pose\0.5.1675469404`? The files are automatically downloaded by webui.sh/webui.bat.

Yes, but this file is only 15Mb and there are no changes in the download

**Comment by wsjhou** [comment]
> @wsjhou谢谢你。Chrome开发者工具的控制台或网络面板有红色错误吗？ ![image](https://user-images.githubusercontent.com/42905588/230131791-4bf71135-eec4-4311-8b56-1af0e74cd2e6.png)

Thank you very much! I use Google browser loading can, the original browser is the problem, never thought about it before is really embarrassing

**Comment by wsjhou** [comment]
> 我也是！



> 我也是！

I fixed it today, my problem was probably due to browser incompatibility, maybe you can try it?

**Comment by gtbloody** [comment]
> @wsjhou Thank you. Are there any red errors in the Console or Network panel of the Chrome Developer Tool? ![image](https://user-images.githubusercontent.com/42905588/230131791-4bf71135-eec4-4311-8b56-1af0e74cd2e6.png)

![image](https://user-images.githubusercontent.com/130019359/230832948-30656d55-ab67-4219-8367-391caf26cc1e.png)

I do have red errors in the Console panel of the Chrome Developer Tool!!! Is it a network problem？I‘ve used my VPN already

**Comment by DESEOUMAIGA** [comment]
Got same error, did anyone solve it?

**Comment by feifanlee** [comment]
same error +1
